loopchain/consensus/syncer.py

```diff
@@ -0,0 +1,144 @@
+import time
+import asyncio
+from typing import TYPE_CHECKING, Dict
+from loopchain import utils, configure as conf
+from loopchain.blockchain.blocks.v1_0 import Block
+from loopchain.blockchain.votes.v1_0 import BlockVote
+from loopchain.protos import message_code, loopchain_pb2, loopchain_pb2_grpc
+from loopchain.tools.grpc_helper import GRPCHelper
+from loopchain.channel.channel_property import ChannelProperty
+
+from lft.consensus.events import ReceiveDataEvent, ReceiveVoteEvent
+
+if TYPE_CHECKING:
+    from loopchain.peer.block_manager import BlockManager
+    from lft.event import EventSystem
+
+
+class Syncer:
+    def __init__(self, block_manager: 'BlockManager', event_system: 'EventSystem'):
+        self._block_manager = block_manager
+        self.__blockchain = self._block_manager.blockchain
+        self._event_system: 'EventSystem' = event_system
+        self._data_info_other_nodes: Dict[int, list] = dict()
+        self._vote_info_other_nodes: Dict[int, list] = dict()
+        self._request_history_list = dict()
+        self._target_idx = 0
+        self._stub_list = list()
+
+        self._last_block_height = -1
+        if self.__blockchain.last_block:
+            self._last_block_height = self.__blockchain.last_block.header.height
```

Comment:
There is the property for getting the height of the last block in `blockchain.py`. How about using that?